File: 백준/Bronze/10101. 삼각형 외우기/삼각형 외우기.java
import java.io.BufferedReader;
import java.io.IOException;
import java.io.InputStreamReader;



public class Main {

	public static void main(String[] args) throws IOException {

		BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
		int N = Integer.parseInt(br.readLine());
		int M = Integer.parseInt(br.readLine());
		int O = Integer.parseInt(br.readLine());
		int sumS = N + M + O;
		if(N == 60 && M == 60 && O ==60) {
			System.out.println("Equilateral");
		}
		else if(sumS == 180) {
			if( N == M || N == O || M == O) {
				System.out.println("Isosceles");
			} else {
				System.out.println("Scalene");
			}
		}else {
			System.out.println("Error");
		}
	}

}

File: 백준/Bronze/10101. 삼각형 외우기/삼각형 외우기.py
a = int(input())
b = int(input())
c = int(input())
if a == b == c:
    print('Equilateral')
elif a + b + c == 180:
    if a == b or a == c or b == c:
       print('Isosceles')
    elif a != b!= c:
        print('Scalene')    
elif a + b + c != 180:
    print('Error')        